File: redome-front/src/app/paciente/busca/analise/analise.match.dto.ts
import { BuscaChecklistDTO } from 'app/shared/component/listamatch/classes/busca.checklist.dto';
import { BaseEntidade } from '../../../shared/base.entidade';
import { MatchDTO } from "../../../shared/component/listamatch/match.dto";
import { CentroTransplante } from '../../../shared/dominio/centro.transplante';

/**
 * Classe que representa o dto com a listade matchs de um paciente
 *
 * @author Bruno Sousa
 * @export
 * @class AnaliseMatchDTO
 */
export class AnaliseMatchDTO extends BaseEntidade {

    private _rmr: number;
    private _listaFase1: MatchDTO[];
    private _listaFase2: MatchDTO[];
    private _listaFase3: MatchDTO[];
	private _listaDisponibilidade: MatchDTO[];
	private _abo:string;

	private _buscaId: number;
	private _temPrescricao:boolean;
	private _peso: number;
	private _centroTransplanteConfirmado: CentroTransplante;

	private _totalMedula: number;
	private _totalCordao: number;
	private _totalSolicitacaoMedula: number;
	private _totalSolicitacaoCordao: number;

	private _boolAceitaMismatch:boolean ;
	private _aceitaMismatch:string ;
	private _locusMismatch:string ;
	private _arquivoAnticorpo:string;
	private _transplantePrevio:string;


	private _temExameAnticorpo:string;
	private _resultadoExameAnticorpo:string ;

	private _buscaChecklist: BuscaChecklistDTO[];

	private _boolTemExameAnticorpo:boolean;
	private _dataExameAnticorpo:string;

	private _totalHistoricoFase1: number;
	private _totalHistoricoFase2: number;
	private _totalHistoricoFase3: number;

	private _nomeMedicoResponsavel: string;
	private _nomeCentroAvaliador: string;

	private _doadoresPrescritos: string;
	private _quantidadePrescricoes: number;
	private _tipoDoadorComPrescricao: number;

  private _qtdMatchWmdaMedula: number;
	private _qtdMatchWmdaMedulaImportado: number;
	private _qtdMatchWmdaCordao: number;
	private _qtdMatchWmdaCordaoImportado: number;

	public get rmr(): number {
		return this._rmr;
	}

	public set rmr(value: number) {
		this._rmr = value;
	}

	public get listaFase1(): MatchDTO[] {
		return this._listaFase1;
	}

	public set listaFase1(value: MatchDTO[]) {
		this._listaFase1 = value;
	}

	public get listaFase2(): MatchDTO[] {
		return this._listaFase2;
	}

	public set listaFase2(value: MatchDTO[]) {
		this._listaFase2 = value;
	}


	public get listaFase3(): MatchDTO[] {
		return this._listaFase3;
	}

	public set listaFase3(value: MatchDTO[]) {
		this._listaFase3 = value;
	}

	public get listaDisponibilidade(): MatchDTO[] {
		return this._listaDisponibilidade;
	}

	public set listaDisponibilidade(value: MatchDTO[]) {
		this._listaDisponibilidade = value;
	}

	public get buscaId(): number {
		return this._buscaId;
	}
	public get temPrescricao(): boolean {
		return this._temPrescricao;
	}

	public set buscaId(value: number) {
		this._buscaId = value;
	}

	public set temPrescricao(value: boolean) {
		this._temPrescricao = value;
	}

	public get peso(): number {
		return this._peso;
	}

	public set peso(value: number) {
		this._peso = value;
	}

    /**
     * Indica o centro de transplante confirmado para a busca.
     * @return {CentroTransplante}
     */
	public get centroTransplanteConfirmado(): CentroTransplante {
		return this._centroTransplanteConfirmado;
	}

	public set centroTransplanteConfirmado(value: CentroTransplante) {
		this._centroTransplanteConfirmado = value;
	}


    /**
     * Getter abo
     * @return {string}
     */
	public get abo(): string {
		return this._abo;
	}

    /**
     * Setter abo
     * @param {string} value
     */
	public set abo(value: string) {
		this._abo = value;
	}

    /**
     * Getter totalMedula
     * @return {number}
     */
	public get totalMedula(): number {
		return this._totalMedula;
	}

    /**
     * Setter totalMedula
     * @param {number} value
     */
	public set totalMedula(value: number) {
		this._totalMedula = value;
	}

    /**
     * Getter totalCordao
     * @return {number}
     */
	public get totalCordao(): number {
		return this._totalCordao;
	}

    /**
     * Setter totalCordao
     * @param {number} value
     */
	public set totalCordao(value: number) {
		this._totalCordao = value;
	}


    /**
     * Getter aceitaMismatch
     * @return {string }
     */
	public get aceitaMismatch(): string  {
		return this._aceitaMismatch;
	}

    /**
     * Setter aceitaMismatch
     * @param {string } value
     */
	public set aceitaMismatch(value: string ) {
		this._aceitaMismatch = value;
	}

    /**
     * Getter locusMismatch
     * @return {string }
     */
	public get locusMismatch(): string  {
		return this._locusMismatch;
	}

    /**
     * Setter locusMismatch
     * @param {string } value
     */
	public set locusMismatch(value: string ) {
		this._locusMismatch = value;
	}


    /**
     * Getter arquivoAnticorpo
     * @return {string}
     */
	public get arquivoAnticorpo(): string {
		return this._arquivoAnticorpo;
	}

    /**
     * Setter arquivoAnticorpo
     * @param {string} value
     */
	public set arquivoAnticorpo(value: string) {
		this._arquivoAnticorpo = value;
	}


    /**
     * Getter transplantePrevio
     * @return {string}
     */
	public get transplantePrevio(): string {
		return this._transplantePrevio;
	}

    /**
     * Setter transplantePrevio
     * @param {string} value
     */
	public set transplantePrevio(value: string) {
		this._transplantePrevio = value;
	}

    /**
     * Getter boolAceitaMismatch
     * @return {boolean }
     */
	public get boolAceitaMismatch(): boolean  {
		return this._boolAceitaMismatch;
	}

    /**
     * Setter boolAceitaMismatch
     * @param {boolean } value
     */
	public set boolAceitaMismatch(value: boolean ) {
		this._boolAceitaMismatch = value;
	}


    /**
     * Getter temExameAnticorpo
     * @return {string}
     */
	public get temExameAnticorpo(): string {
		return this._temExameAnticorpo;
	}

    /**
     * Setter temExameAnticorpo
     * @param {string} value
     */
	public set temExameAnticorpo(value: string) {
		this._temExameAnticorpo = value;
	}


    /**
     * Getter resultadoExameAnticorpo
     * @return {string }
     */
	public get resultadoExameAnticorpo(): string  {
		return this._resultadoExameAnticorpo;
	}

    /**
     * Setter resultadoExameAnticorpo
     * @param {string } value
     */
	public set resultadoExameAnticorpo(value: string ) {
		this._resultadoExameAnticorpo = value;
	}

	get doadoresPrescritos(): string {
		return this._doadoresPrescritos;
	}

	set doadoresPrescritos(value: string) {
		this._doadoresPrescritos = value;
	}

	/**
     * Getter boolTemExameAnticorpo
     * @return {boolean}
     */
	public get boolTemExameAnticorpo(): boolean {
		return this._boolTemExameAnticorpo;
	}

    /**
     * Setter boolTemExameAnticorpo
     * @param {boolean} value
     */
	public set boolTemExameAnticorpo(value: boolean) {
		this._boolTemExameAnticorpo = value;
	}

	public get buscaChecklist(): BuscaChecklistDTO[] {
		return this._buscaChecklist;
	}

	public set buscaChecklist(value: BuscaChecklistDTO[]) {
		this._buscaChecklist = value;
	}


    /**
     * Getter dataExameAnticorpo
     * @return {string}
     */
	public get dataExameAnticorpo(): string {
		return this._dataExameAnticorpo;
	}

    /**
     * Setter dataExameAnticorpo
     * @param {string} value
     */
	public set dataExameAnticorpo(value: string) {
		this._dataExameAnticorpo = value;
	}

    /**
     * Getter totalHistoricoFase1
     * @return {number}
     */
	public get totalHistoricoFase1(): number {
		return this._totalHistoricoFase1;
	}

    /**
     * Setter totalHistoricoFase1
     * @param {number} value
     */
	public set totalHistoricoFase1(value: number) {
		this._totalHistoricoFase1 = value;
	}

    /**
     * Getter totalHistoricoFase2
     * @return {number}
     */
	public get totalHistoricoFase2(): number {
		return this._totalHistoricoFase2;
	}

    /**
     * Setter totalHistoricoFase2
     * @param {number} value
     */
	public set totalHistoricoFase2(value: number) {
		this._totalHistoricoFase2 = value;
	}

    /**
     * Getter totalHistoricoFase3
     * @return {number}
     */
	public get totalHistoricoFase3(): number {
		return this._totalHistoricoFase3;
	}

    /**
     * Setter totalHistoricoFase3
     * @param {number} value
     */
	public set totalHistoricoFase3(value: number) {
		this._totalHistoricoFase3 = value;
	}

	public jsonToEntity(res: any) {
        throw new Error("Method not implemented.");
	}

    /**
     * Getter totalSolicitacaoMedula
     * @return {number}
     */
	public get totalSolicitacaoMedula(): number {
		return this._totalSolicitacaoMedula;
	}

    /**
     * Setter totalSolicitacaoMedula
     * @param {number} value
     */
	public set totalSolicitacaoMedula(value: number) {
		this._totalSolicitacaoMedula = value;
	}

    /**
     * Getter totalSolicitacaoCordao
     * @return {number}
     */
	public get totalSolicitacaoCordao(): number {
		return this._totalSolicitacaoCordao;
	}

    /**
     * Setter totalSolicitacaoCordao
     * @param {number} value
     */
	public set totalSolicitacaoCordao(value: number) {
		this._totalSolicitacaoCordao = value;
	}

    /**
     * Getter nomeMedicoResponsavel
     * @return {string}
     */
	public get nomeMedicoResponsavel(): string {
		return this._nomeMedicoResponsavel;
	}

    /**
     * Setter nomeMedicoResponsavel
     * @param {string} value
     */
	public set nomeMedicoResponsavel(value: string) {
		this._nomeMedicoResponsavel = value;
	}

    /**
     * Getter nomeCentroAvaliador
     * @return {string}
     */
	public get nomeCentroAvaliador(): string {
		return this._nomeCentroAvaliador;
	}

    /**
     * Setter nomeCentroAvaliador
     * @param {string} value
     */
	public set nomeCentroAvaliador(value: string) {
		this._nomeCentroAvaliador = value;
	}

	public get quantidadePrescricoes(): number {
		return this._quantidadePrescricoes;
	}

	public set quantidadePrescricoes(value: number) {
		this._quantidadePrescricoes = value;
	}

	get tipoDoadorComPrescricao(): number {
		return this._tipoDoadorComPrescricao;
	}

	set tipoDoadorComPrescricao(value: number) {
		this._tipoDoadorComPrescricao = value;
  }


    /**
     * Getter qtdMatchWmdaMedula
     * @return {number}
     */
	public get qtdMatchWmdaMedula(): number {
		return this._qtdMatchWmdaMedula;
	}

    /**
     * Setter qtdMatchWmdaMedula
     * @param {number} value
     */
	public set qtdMatchWmdaMedula(value: number) {
		this._qtdMatchWmdaMedula = value;
	}

    /**
     * Getter qtdMatchWmdaMedulaImportado
     * @return {number}
     */
	public get qtdMatchWmdaMedulaImportado(): number {
		return this._qtdMatchWmdaMedulaImportado;
	}

    /**
     * Setter qtdMatchWmdaMedulaImportado
     * @param {number} value
     */
	public set qtdMatchWmdaMedulaImportado(value: number) {
		this._qtdMatchWmdaMedulaImportado = value;
	}


    /**
     * Getter qtdMatchWmdaCordao
     * @return {number}
     */
	public get qtdMatchWmdaCordao(): number {
		return this._qtdMatchWmdaCordao;
	}

    /**
     * Setter qtdMatchWmdaCordao
     * @param {number} value
     */
	public set qtdMatchWmdaCordao(value: number) {
		this._qtdMatchWmdaCordao = value;
	}


    /**
     * Getter qtdMatchWmdaCordaoImportado
     * @return {number}
     */
	public get qtdMatchWmdaCordaoImportado(): number {
		return this._qtdMatchWmdaCordaoImportado;
	}

    /**
     * Setter qtdMatchWmdaCordaoImportado
     * @param {number} value
     */
	public set qtdMatchWmdaCordaoImportado(value: number) {
		this._qtdMatchWmdaCordaoImportado = value;
	}

}

File: redome/src/main/java/br/org/cancer/modred/controller/dto/AnaliseMatchDTO.java
package br.org.cancer.modred.controller.dto;

import java.math.BigDecimal;
import java.util.ArrayList;
import java.util.List;

import br.org.cancer.modred.model.CentroTransplante;

/**
 * Classe de DTO para informações com as listas de matchs.
 * 
 * @author Fillipe Queiroz
 */
public class AnaliseMatchDTO {

	private Long rmr;
	private Long buscaId;
	private BigDecimal peso; 
	private List<MatchDTO> listaFase1;
	private List<MatchDTO> listaFase2;
	private List<MatchDTO> listaFase3;
	private List<MatchDTO> listaExameFase2;
	private List<MatchDTO> listaExameFase3;
	private List<MatchDTO> listaDisponibilidade;
	private Boolean temPrescricao;	
	private CentroTransplante centroTransplanteConfirmado;
	private String abo;
	private Long totalMedula;
	private Long totalCordao;
	private Long totalSolicitacaoMedula;
	private Long totalSolicitacaoCordao;
	private Boolean aceitaMismatch;
	private String locusMismatch;
	private Boolean temExameAnticorpo;
	private String dataAnticorpo;
	private String resultadoExameAnticorpo;
	private String transplantePrevio;
	private String doadoresPrescritos;
	private List<BuscaChecklistDTO> buscaChecklist;
	private Long totalHistoricoFase1;
	private Long totalHistoricoFase2;
	private Long totalHistoricoFase3;
	
	private String nomeMedicoResponsavel;
	private String nomeCentroAvaliador;
	 
	private Long quantidadePrescricoes;
	private Long tipoDoadorComPrescricao;
	
	private Integer qtdMatchWmdaMedula = 0;
	private Integer qtdMatchWmdaMedulaImportado = 0;
	private Integer qtdMatchWmdaCordao = 0;
	private Integer qtdMatchWmdaCordaoImportado = 0;

	
	/**
	 * @return the rmr
	 */
	public Long getRmr() {
		return rmr;
	}

	/**
	 * @param rmr the rmr to set
	 */
	public void setRmr(Long rmr) {
		this.rmr = rmr;
	}

	/**
	 * @return the listaFase1
	 */
	public List<MatchDTO> getListaFase1() {
		return listaFase1;
	}

	/**
	 * @param listaFase1 the listaFase1 to set
	 */
	public void setListaFase1(List<MatchDTO> listaFase1) {
		this.listaFase1 = listaFase1;
	}

	/**
	 * @return the listaFase2
	 */
	public List<MatchDTO> getListaFase2() {
		return listaFase2;
	}

	/**
	 * @param listaFase2 the listaFase2 to set
	 */
	public void setListaFase2(List<MatchDTO> listaFase2) {
		this.listaFase2 = listaFase2;
	}

	/**
	 * @return the listaFase3
	 */
	public List<MatchDTO> getListaFase3() {
		return listaFase3;
	}

	/**
	 * @param listaFase3 the listaFase3 to set
	 */
	public void setListaFase3(List<MatchDTO> listaFase3) {
		this.listaFase3 = listaFase3;
	}

	/**
	 * @return the listaExameFase2
	 */
	public List<MatchDTO> getListaExameFase2() {
		return listaExameFase2;
	}

	/**
	 * @param listaExameFase2 the listaExameFase2 to set
	 */
	public void setListaExameFase2(List<MatchDTO> listaExameFase2) {
		this.listaExameFase2 = listaExameFase2;
	}

	/**
	 * @return the listaExameFase3
	 */
	public List<MatchDTO> getListaExameFase3() {
		return listaExameFase3;
	}

	/**
	 * @param listaExameFase3 the listaExameFase3 to set
	 */
	public void setListaExameFase3(List<MatchDTO> listaExameFase3) {
		this.listaExameFase3 = listaExameFase3;
	}

	/**
	 * @return the listaDisponibilidade
	 */
	public List<MatchDTO> getListaDisponibilidade() {
		return listaDisponibilidade;
	}

	/**
	 * @param listaDisponibilidade the listaDisponibilidade to set
	 */
	public void setListaDisponibilidade(List<MatchDTO> listaDisponibilidade) {
		this.listaDisponibilidade = listaDisponibilidade;
	}

	public Long getBuscaId() {
		return buscaId;
	}

	public void setBuscaId(Long buscaId) {
		this.buscaId = buscaId;
	}

	/**
	 * @return lista com todas as listas
	 */
	public List<MatchDTO> getAllLists() {
		List<MatchDTO> allLists = new ArrayList<MatchDTO>();
		if(listaFase1 != null){
			allLists.addAll(listaFase1);
		}
		if(listaFase2 != null){
			allLists.addAll(listaFase2);
		}
		if(listaFase3 != null){
			allLists.addAll(listaFase3);
		}
		if(listaExameFase2 != null){
			allLists.addAll(listaExameFase2);
		}
		if(listaExameFase3 != null){
			allLists.addAll(listaExameFase3);
		}
		if(listaDisponibilidade != null){
			allLists.addAll(listaDisponibilidade);
		}

		return allLists;
	}

	/**
	 * @return the temPrescricao
	 */
	public Boolean getTemPrescricao() {
		return temPrescricao;
	}

	/**
	 * @param temPrescricao the temPrescricao to set
	 */
	public void setTemPrescricao(Boolean temPrescricao) {
		this.temPrescricao = temPrescricao;
	}

	
	/**
	 * @return the peso
	 */
	public BigDecimal getPeso() {
		return peso;
	}

	
	/**
	 * @param peso the peso to set
	 */
	public void setPeso(BigDecimal peso) {
		this.peso = peso;
	}

	/**
	 * O centro de transplante confirmado para o paciente.
	 * 
	 * @return centro de transplante associado a busca.
	 */
	public CentroTransplante getCentroTransplanteConfirmado() {
		return centroTransplanteConfirmado;
	}

	public void setCentroTransplanteConfirmado(CentroTransplante centroTransplanteConfirmado) {
		this.centroTransplanteConfirmado = centroTransplanteConfirmado;
	}

	
	/**
	 * @return the abo
	 */
	public String getAbo() {
		return abo;
	}

	
	/**
	 * @param abo the abo to set
	 */
	public void setAbo(String abo) {
		this.abo = abo;
	}

	
	/**
	 * @return the totalMedula
	 */
	public Long getTotalMedula() {
		return totalMedula;
	}

	
	/**
	 * @param totalMedula the totalMedula to set
	 */
	public void setTotalMedula(Long totalMedula) {
		this.totalMedula = totalMedula;
	}

	
	/**
	 * @return the totalCordao
	 */
	public Long getTotalCordao() {
		return totalCordao;
	}

	
	/**
	 * @param totalCordao the totalCordao to set
	 */
	public void setTotalCordao(Long totalCordao) {
		this.totalCordao = totalCordao;
	}

	/**
	 * @return the aceitaMismatch
	 */
	public Boolean getAceitaMismatch() {
		return aceitaMismatch;
	}

	/**
	 * @param aceitaMismatch the aceitaMismatch to set
	 */
	public void setAceitaMismatch(Boolean aceitaMismatch) {
		this.aceitaMismatch = aceitaMismatch;
	}

	/**
	 * @return the locusMismatch
	 */
	public String getLocusMismatch() {
		return locusMismatch;
	}

	/**
	 * @param locusMismatch the locusMismatch to set
	 */
	public void setLocusMismatch(String locusMismatch) {
		this.locusMismatch = locusMismatch;
	}

	
	
	

	/**
	 * @return the temExameAnticorpo
	 */
	public Boolean getTemExameAnticorpo() {
		return temExameAnticorpo;
	}

	/**
	 * @param temExameAnticorpo the temExameAnticorpo to set
	 */
	public void setTemExameAnticorpo(Boolean temExameAnticorpo) {
		this.temExameAnticorpo = temExameAnticorpo;
	}
	
	

	/**
	 * @return the dataAnticorpo
	 */
	public String getDataAnticorpo() {
		return dataAnticorpo;
	}

	/**
	 * @param dataAnticorpo the dataAnticorpo to set
	 */
	public void setDataAnticorpo(String dataAnticorpo) {
		this.dataAnticorpo = dataAnticorpo;
	}

	/**
	 * @return the resultadoExameAnticorpo
	 */
	public String getResultadoExameAnticorpo() {
		return resultadoExameAnticorpo;
	}

	/**
	 * @param resultadoExameAnticorpo the resultadoExameAnticorpo to set
	 */
	public void setResultadoExameAnticorpo(String resultadoExameAnticorpo) {
		this.resultadoExameAnticorpo = resultadoExameAnticorpo;
	}

	/**
	 * @return the transplantePrevio
	 */
	public String getTransplantePrevio() {
		return transplantePrevio;
	}

	/**
	 * @param transplantePrevio the transplantePrevio to set
	 */
	public void setTransplantePrevio(String transplantePrevio) {
		this.transplantePrevio = transplantePrevio;
	}
	
	/**
	 * @return the doadoresPrescritos
	 */
	public String getDoadoresPrescritos() {
		return doadoresPrescritos;
	}

	/**
	 * @param doadoresPrescritos the doadoresPrescritos to set
	 */
	public void setDoadoresPrescritos(String doadoresPrescritos) {
		this.doadoresPrescritos = doadoresPrescritos;
	}

	/**
	 * Lista de itens de checklist a serem exibidos para o paciente.
	 * 
	 * @return lista de DTOs contendo a descrição e prazo do ocorrido.
	 */
	public List<BuscaChecklistDTO> getBuscaChecklist() {
		return buscaChecklist;
	}

	public void setBuscaChecklist(List<BuscaChecklistDTO> buscaChecklist) {
		this.buscaChecklist = buscaChecklist;
	}

	/**
	 * @return the totalHistoricoFase1
	 */
	public Long getTotalHistoricoFase1() {
		return totalHistoricoFase1;
	}

	/**
	 * @param totalHistoricoFase1 the totalHistoricoFase1 to set
	 */
	public void setTotalHistoricoFase1(Long totalHistoricoFase1) {
		this.totalHistoricoFase1 = totalHistoricoFase1;
	}

	/**
	 * @return the totalHistoricoFase2
	 */
	public Long getTotalHistoricoFase2() {
		return totalHistoricoFase2;
	}

	/**
	 * @param totalHistoricoFase2 the totalHistoricoFase2 to set
	 */
	public void setTotalHistoricoFase2(Long totalHistoricoFase2) {
		this.totalHistoricoFase2 = totalHistoricoFase2;
	}

	/**
	 * @return the totalHistoricoFase3
	 */
	public Long getTotalHistoricoFase3() {
		return totalHistoricoFase3;
	}

	/**
	 * @param totalHistoricoFase3 the totalHistoricoFase3 to set
	 */
	public void setTotalHistoricoFase3(Long totalHistoricoFase3) {
		this.totalHistoricoFase3 = totalHistoricoFase3;
	}

	/**
	 * @return the totalSolicitacaoMedula
	 */
	public Long getTotalSolicitacaoMedula() {
		return totalSolicitacaoMedula;
	}

	/**
	 * @param totalSolicitacaoMedula the totalSolicitacaoMedula to set
	 */
	public void setTotalSolicitacaoMedula(Long totalSolicitacaoMedula) {
		this.totalSolicitacaoMedula = totalSolicitacaoMedula;
	}

	/**
	 * @return the totalSolicitacaoCordao
	 */
	public Long getTotalSolicitacaoCordao() {
		return totalSolicitacaoCordao;
	}

	/**
	 * @param totalSolicitacaoCordao the totalSolicitacaoCordao to set
	 */
	public void setTotalSolicitacaoCordao(Long totalSolicitacaoCordao) {
		this.totalSolicitacaoCordao = totalSolicitacaoCordao;
	}

	/**
	 * @return the nomeMedicoResponsavel
	 */
	public String getNomeMedicoResponsavel() {
		return nomeMedicoResponsavel;
	}

	/**
	 * @param nomeMedicoResponsavel the nomeMedicoResponsavel to set
	 */
	public void setNomeMedicoResponsavel(String nomeMedicoResponsavel) {
		this.nomeMedicoResponsavel = nomeMedicoResponsavel;
	}

	/**
	 * @return the nomeCentroAvaliador
	 */
	public String getNomeCentroAvaliador() {
		return nomeCentroAvaliador;
	}

	/**
	 * @param nomeCentroAvaliador the nomeCentroAvaliador to set
	 */
	public void setNomeCentroAvaliador(String nomeCentroAvaliador) {
		this.nomeCentroAvaliador = nomeCentroAvaliador;
	}

	/**
	 * @return the quantidadePrescricoes
	 */
	public Long getQuantidadePrescricoes() {
		return quantidadePrescricoes;
	}

	/**
	 * @param quantidadePrescricoes the quantidadePrescricoes to set
	 */
	public void setQuantidadePrescricoes(Long quantidadePrescricoes) {
		this.quantidadePrescricoes = quantidadePrescricoes;
	}

	/**
	 * @return the tipoDoadorComPrescricao
	 */
	public Long getTipoDoadorComPrescricao() {
		return tipoDoadorComPrescricao;
	}

	/**
	 * @param tipoDoadorComPrescricao the tipoDoadorComPrescricao to set
	 */
	public void setTipoDoadorComPrescricao(Long tipoDoadorComPrescricao) {
		this.tipoDoadorComPrescricao = tipoDoadorComPrescricao;
	}

	/**
	 * @return the qtdMatchWmdaMedula
	 */
	public Integer getQtdMatchWmdaMedula() {
		return qtdMatchWmdaMedula;
	}

	/**
	 * @param qtdMatchWmdaMedula the qtdMatchWmdaMedula to set
	 */
	public void setQtdMatchWmdaMedula(Integer qtdMatchWmdaMedula) {
		this.qtdMatchWmdaMedula = qtdMatchWmdaMedula;
	}

	/**
	 * @return the qtdMatchWmdaMedulaImportado
	 */
	public Integer getQtdMatchWmdaMedulaImportado() {
		return qtdMatchWmdaMedulaImportado;
	}

	/**
	 * @param qtdMatchWmdaMedulaImportado the qtdMatchWmdaMedulaImportado to set
	 */
	public void setQtdMatchWmdaMedulaImportado(Integer qtdMatchWmdaMedulaImportado) {
		this.qtdMatchWmdaMedulaImportado = qtdMatchWmdaMedulaImportado;
	}

	/**
	 * @return the qtdMatchWmdaCordao
	 */
	public Integer getQtdMatchWmdaCordao() {
		return qtdMatchWmdaCordao;
	}

	/**
	 * @param qtdMatchWmdaCordao the qtdMatchWmdaCordao to set
	 */
	public void setQtdMatchWmdaCordao(Integer qtdMatchWmdaCordao) {
		this.qtdMatchWmdaCordao = qtdMatchWmdaCordao;
	}

	/**
	 * @return the qtdMatchWmdaCordaoImportado
	 */
	public Integer getQtdMatchWmdaCordaoImportado() {
		return qtdMatchWmdaCordaoImportado;
	}

	/**
	 * @param qtdMatchWmdaCordaoImportado the qtdMatchWmdaCordaoImportado to set
	 */
	public void setQtdMatchWmdaCordaoImportado(Integer qtdMatchWmdaCordaoImportado) {
		this.qtdMatchWmdaCordaoImportado = qtdMatchWmdaCordaoImportado;
	}
	
}
